.github/workflows/ci.yaml

```diff
@@ -55,7 +63,7 @@ jobs:
 
     integration-tests-postgres:
         name: PostgreSQL integration tests
-        needs: tests
+        # needs: tests
```

Comment:
We should remember to revert this one (although technically it's OSS, so we could run integration tests against databases immediately - github pays :dollar: )